### PR TITLE
Decrease Steps to Deployment - Login

### LINKED ISSUE #879: Decrease manual steps for deploy
Currently we do have a manual step to setup the azure devops configuration (organization and project). There is the same code included on the post-create.sh script.  We need to check if this configuration can be automated and if so, remove the manual step from the ReadMe guidance.

**DoD**
- Test if this command az devops configure --defaults organization="$AZDO_ORGANIZATION_URL" project="$AZDO_PROJECT" can be automated
- If so, delete the manual step from the ReadMe

### PR DESCRIPTION
**Pull request by LiliamLeme**
# Type of PR
Code Changes, Documentation Changes

## Purpose

1) init_environment .sh - Change the init_environment to also comtemplate Login
2) post_create.sh - Remove Azdo configuration from the file
3)Deploy.sh - Adding the init_enviromnent to run before the 
prequirements
4)change Readme to reflect those changes in Login

## Author pre-publish checklist

- [ X] Added test to prove my fix is effective or new feature works
- [ ] No PII in logs
- [X ] Made corresponding changes to the documentation

## Validation steps

- ...Test checking the logging from the deployment
- 
https://aka.ms/CLI_refstatus
WARNING: A web browser has been opened at https://login.microsoftonline.com/xxx/oauth2/v2.0/authorize. Please continue the login in the web browser. If no web browser is available or if the web browser fails to open, use device code flow with `az login --use-device-code`.
[
  {
    "cloudName": "AzureCloud",
image
Scenarios:

1) Interrupt the deployment in the middle, re-started the deployment

2) Rebuild container - Restarted and it is ok.

## Issues Closed or Referenced

- Closes #879 

- References #issue_number

### CODE REVIEW

**Review by thesqlpro — APPROVED** [review]
Looks good

File: e2e_samples/parking_sensors/README.md [hunk 1]
```diff
@@ -291,46 +291,23 @@ Set up the environment variables as specified, fork the GitHub repository, and l
...
-        az devops configure --defaults organization=$AZDO_ORGANIZATION_URL project=$AZDO_PROJECT
-        ```
+- Ensure that you have completed the configuration for the variables described in the previous section, titled **Configuration: Variables and Login**.
 
+  - This configuration will be used during the environment deployment process to facilitate login.
   - Create a `cluster.config.json` Spark configuration from the [`cluster.config.template.json`](./databricks/config/cluster.config.template.json) file. For the "node_type_id" field, select a SKU that is available from the following command in your subscription:
 
-        ```bash
-        az vm list-usage --location "<YOUR_REGION>" -o table
+  -     ```bash
```
Comment (ydaponte) [inline e2e_samples/parking_sensors/README.md:301]: I see you were trying to fix the indentation of the feature branch but  if you preview the file on your branch shows like bellow - is still not fixed. Can you please fix it? In the main branch it is showing still correct if you want to copy from there.
image

File: e2e_samples/parking_sensors/scripts/init_environment.sh [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+
+if az account show > /dev/null 2>&1; then
+    echo "Already logged in. Logging out and logging in again."
+    az logout
+fi
+
+az config set core.login_experience_v2=off
+az login --tenant $TENANT_ID
+az config set core.login_experience_v2=on
+
```
Comment (ydaponte) [inline e2e_samples/parking_sensors/scripts/init_environment.sh:20]:
```suggestion
```
Getting rid of the white lines

File: e2e_samples/parking_sensors/scripts/init_environment.sh [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+az account set -s $AZURE_SUBSCRIPTION_ID
+
```
Comment (ydaponte) [inline e2e_samples/parking_sensors/scripts/init_environment.sh:22]:
```suggestion
```
Getting rid of the white lines

File: e2e_samples/parking_sensors/scripts/init_environment.sh [hunk 4]
```diff
@@ -1,5 +1,30 @@
...
+
```
Comment (ydaponte) [inline e2e_samples/parking_sensors/scripts/init_environment.sh:23]:
```suggestion
```
Getting rid of the white lines

File: e2e_samples/parking_sensors/scripts/init_environment.sh [hunk 5]
```diff
@@ -1,5 +1,30 @@
...
+
+az devops configure --defaults organization=$AZDO_ORGANIZATION_URL project=$AZDO_PROJECT
+
```
Comment (ydaponte) [inline e2e_samples/parking_sensors/scripts/init_environment.sh:26]:
```suggestion
```
Getting rid of the white lines

**Review by ydaponte — CHANGES_REQUESTED** [review]
Minor issues with indentation, but they need to be fixed before the merge. Also added suggestion just to get rid of some white lines that are not necessary - you can just easily accept the suggestion from the PR to be easier. Thank you!

File: e2e_samples/parking_sensors/README.md [hunk 6]
```diff
@@ -291,46 +291,23 @@ Set up the environment variables as specified, fork the GitHub repository, and l
    - To further customize the solution, set parameters in `arm.parameters` files located in the `infrastructure` folder.
       - To enable Observability and Monitoring components through code(Observability-as-code), please set enable_monitoring parameter to true in  `arm.parameters` files located in the `infrastructure` folder. This will deploy log analytics workspace to collect monitoring data from key resources, setup an Azure dashboards to monitor key metrics and configure alerts for ADF pipelines.
   
-  **Login**
+  **Login and Cluster Configuration**
```
[the remark this answers is not in the document]
Comment (LiliamLeme) [reply e2e_samples/parking_sensors/README.md]: it should be aligned with step 1. Done... let me know if it is more clear

File: e2e_samples/parking_sensors/README.md (hunk 1 above)
Comment (LiliamLeme) [reply e2e_samples/parking_sensors/README.md]: definitely it was not right. Fixed

**Review by ydaponte — APPROVED** [review]
Great! Thanks for making the changes!

**Comment by LiliamLeme** [comment]
I added the variable change I mentioned in the meeting:
image
@thesqlpro , @ydaponte , @elenaterenzi

**Review by ydaponte — APPROVED** [review]
Good addition! Approving!

**Pull request merged**